src/postgres/tree/PostgreSQLAccountTreeItem.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { DatabaseListResult, Server } from 'azure-arm-postgresql/lib/models';
+import { Databases } from 'azure-arm-postgresql/lib/operations';
+import * as vscode from 'vscode';
+import { AzureParentTreeItem, AzureTreeItem, ISubscriptionContext } from 'vscode-azureextensionui';
+import { getThemeAgnosticIconPath } from '../../constants';
+import { PostgreSQLDatabaseTreeItem } from './PostgreSQLDatabaseTreeItem';
+
+export class PostgreSQLAccountTreeItem extends AzureParentTreeItem<ISubscriptionContext> {
+    public static contextValue: string = "cosmosDBPostgresServer";
+    public readonly contextValue: string = PostgreSQLAccountTreeItem.contextValue;
+    public readonly childTypeLabel: string = "Database";
+    public readonly id: string;
+    public readonly label: string;
+    public readonly connectionString: string;
+    public readonly host: string;
+    public readonly databases: Databases;
```

Comment:
PostgreSQLManagementClient is initialized in Subscription...Tree already and is used only once to get the server object and the databases object. I feel like its redundant to use PostgreSQLManagementClient again in this file. If we are trying to reduce the number of property variables, we can just pass client instead of server and databases and then access them later in the file as needed?

Reply:
I fixed this and along with removing initPostgres function from Subscription...Tree.ts, there are only 3 property variables for this file now --> server, databases and label and only 2 parameters for the constructor. Is that better now, or should I pass client instead of both server and databases?

Reply:
After more thought, here's my reasoning:

1. The name "databases" confuses me - I would expect that to be the result of `listDatabases`, not the client itself. Something like "databaseClient" would make more sense, but the type itself is called "Databases" and you can't change that
1. I try to avoid caching things like clients because they deal with credentials. I don't know how it works under the covers, but theoretically there could be a refresh token that only gets updated when you create a new client and thus it could expire if you keep an old client around.
1. That's just how we do it elsewhere in our repos 🤷‍♂️ This is the worst reason in the list, but if either option results in about the same lines of code, it serves as a decent tie-breaker

Reply:
Makes sense. Databases is confusing as a name especially being passed to the PostgreSQLAccount... class. I fixed it with only passing client and the server to the PostgresSQLAccount... and handling databases object in the loadMoreChildren... method.